perfscale_regression_ci/scripts/scalability/node-affinity-anti-affinity.sh

```diff
@@ -0,0 +1,171 @@
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-18082
+## Cluster config: Cluster needed in AWS EC2 (m5.xlarge, 3 master/etcd, 3 worker/compute nodes)
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/node-affinity-anti-affinity-config.yml
+################################################ 
+
+
+source ../../utils/run_workload.sh
+source ../custom_workload_env.sh
+source ../common.sh
+source node-affinity-anti-affinity_env.sh
+
+function show_node_labels() {
+  oc get node --show-labels
+  oc get node -l cpu=4
+  oc get node -l cpu=6
+  oc get node -l beta.kubernetes.io/arch=intel
+}
+
+function check_no_error_pods()
+{
+  error=`oc get pods -n $1 | grep Error | wc -l`
+  if [ $error -ne 0 ]; then
+    echo "$error pods found, exiting"
+    #loop to find logs of error pods?
+    exit 1
+  fi
+}
+
+export ANTI_AFFINITY_JOB_ITERATION=${ANTI_AFFINITY_JOB_ITERATION:-130}
+export AFFINITY_JOB_ITERATION=${AFFINITY_JOB_ITERATION:-200}
+
+# Output some general information about the test environment
+date
+uname -a
+oc get clusterversion
+oc version
+oc get node --show-labels
+oc describe node | grep Runtime
+
+
+echo "======Setup/Configuration: label nodes for Affinity and anti-affinity scheduling======"
+
+compute_nodes=$(oc get nodes -l 'node-role.kubernetes.io/worker=' | awk '{print $1}' | grep -v NAME | xargs)
+
+echo -e "\nWorker  nodes are: $compute_nodes"
+
+declare -a node_array
+counter=1
+
+oc get nodes -l 'node-role.kubernetes.io/worker='
+oc describe nodes -l 'node-role.kubernetes.io/worker='
+
+initial_node_label="beta.kubernetes.io/arch=amd64"
+
+
+for n in ${compute_nodes}; do
+  node_array[${counter}]=${n}
+  counter=$((counter+1))
+done
+
+# output node array elements
+for i in {1..2}; do
+  echo "Array element node_array index $i has value : ${node_array[${i}]}"
+done
+
+
+# The following code removes node labels created by the test run and then show the labels have been removed.
+# Uncomment the code when running  this test repeatedly on the same cluster (during debugging) to ensure the 
+# node labels are removed. Otherwise, the test will fail/end prematurely.
+#   echo -e "\nRemoving the node labels"
+# oc label nodes ${node_array[1]} cpu-
+# oc label nodes ${node_array[2]} cpu-
+# oc label nodes ${node_array[1]} --overwrite ${initial_node_label}
+
+
+echo -e "\nLabeling node ${node_array[1]} with label 'cpu=4'"
+oc label nodes ${node_array[1]} cpu=4
+
+echo -e "\nLabeling node ${node_array[2]} with label 'cpu=6'"
+oc label nodes ${node_array[2]} cpu=6
+
+echo -e "\nLabeling node ${node_array[1]} with label 'beta.kubernetes.io/arch=intel'"
+oc label nodes ${node_array[1]} --overwrite beta.kubernetes.io/arch=intel
+
+show_node_labels
+
+sleep 5
+
+
+echo "======Use kube-burner to load the cluster with test objects======"
+run_workload
+
+
+echo "======Waiting for the pods in both namespaces======"
```

Comment:
I think this should be covered in the run_workload. I think it should wait for all the pods to go running before showing success. 

We might need to add in a check of pass/fail after run_workload, think we can do this in separate PR if you don't want to tackle